hypertrace-graphql-entity-schema/src/main/java/org/hypertrace/graphql/entity/request/DefaultEntityRequestBuilder.java

```diff
@@ -102,15 +125,17 @@
             this.timeRange(arguments),
             this.space(arguments),
             this.getOutgoingEdges(selectionSet)),
-        (resultSetRequest, metricRequestList, incomingEdges, outgoingEdges) ->
+        attributeRequestBuilder.buildForKey(context, scope, LABELS_KEY_NAME),
+        (resultSetRequest, metricRequestList, incomingEdges, outgoingEdges, attributeRequest) ->
             new DefaultEntityRequest(
                 scope,
                 resultSetRequest,
                 metricRequestList,
                 incomingEdges,
                 outgoingEdges,
                 includeInactive,
-                fetchTotal));
+                fetchTotal,
+                buildLabelRequest(fetchLabels, attributeRequest)));
```

Comment:
nit: not a big fan of passing booleans to the functions, if can be avoided

```suggestion
                fetchLabels ? Optional.of(buildLabelRequest(attributeRequest)) : Optional.empty();
```

Reply:
for brevity sake, just calling a method with arguments is better. Anyways it just a single line code, so moved it inline.